javascript/javascript2/week1/main.js

```diff
@@ -0,0 +1,16 @@
+console.log('Script loaded');//something is missing I thinks so here,but I written the code according to the requirements.
+const testProductNames=getAvailableProducts();
+console.log(testProductNames[0]);
+const testProductNames=["Laptop","TV","Ipad"];
+function renderProducts(products)
+{
+    const ulTag=document.querySelector('section.products>ul');
+    for(let i=0;i<products.length;i++)
+    {
+        const currentProduct=products[i];
+        const liTag=document.createElement('li');
+        ulTag.appendChild(liTag);
```

Comment:
this needs to build a list of items from the products

each item looks like this
```
{
  "id": "Computer99021",
  "name": "Computer",
  "price": 1130,
  "rating": 8,
  "shipsTo": [
    "Sweden"
  ]
}
```

you should be using `createElement` and `appendChild` to get the data from the objects above and put them into the page :)